This is the in-house patch which changes the build option DUMP_CORE to prevent Samba
to dump the core on internal error conditions (i.e. # of groups > NGROUP_MAX).
...see the original comment which remains in code.

--- a/source3/include/local.h	2014-06-22 23:03:27.000000000 -0700
+++ samba-3.6.24/source3/include/local.h	2014-10-29 10:00:48.268346403 -0700
@@ -142,7 +142,7 @@
 /* do you want to dump core (carefully!) when an internal error is
    encountered? Samba will be careful to make the core file only
    accessible to root */
-#define DUMP_CORE 1
+#define DUMP_CORE 0
 
 /* shall we support browse requests via a FIFO to nmbd? */
 #define ENABLE_FIFO 1
